test/integration/controlplane/transformation/transformation_testcase.go

```diff
@@ -236,10 +236,14 @@ func (e *transformTest) createSecret(name, namespace string) (*corev1.Secret, er
 }
 
 func (e *transformTest) readRawRecordFromETCD(path string) (*clientv3.GetResponse, error) {
-	_, etcdClient, err := integration.GetEtcdClients(e.kubeAPIServer.ServerOpts.Etcd.StorageConfig.Transport)
+	rawClient, etcdClient, err := integration.GetEtcdClients(e.kubeAPIServer.ServerOpts.Etcd.StorageConfig.Transport)
 	if err != nil {
 		return nil, fmt.Errorf("failed to create etcd client: %v", err)
 	}
+	// kvClient is a wrapper around rawClient and to avoid leaking goroutines we need to
```

Comment:
not need to change, but I think this is a copy paste error, right?
since here is called `etcdClient` and in other places `kvClient`

Reply:
Yeah - copy-paste error. Will fix in a subsequent PR.

Reply:
I will be surprised if anybody else will look at this comment in the future :smile: